VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: HMI_System CAVs_System


BO_ 2 Vehicle_Info: 6 Vector__XXX
 SG_ shifter_position : 5|3@1+ (1,0) [0|7] "" Vector__XXX
 SG_ lane_centering_status : 3|2@1+ (1,0) [0|3] "" Vector__XXX
 SG_ lane_right_detected : 2|1@1+ (1,0) [0|1] "" Vector__XXX
 SG_ lane_left_detected : 1|1@1+ (1,0) [0|1] "" Vector__XXX
 SG_ desired_vehicle_pos : 32|16@1- (0.00390625,0) [-127|128] "" Vector__XXX
 SG_ yaw_rate : 16|16@1- (0.00549316406,0) [-180|180] "" Vector__XXX
 SG_ vehicle_on : 0|1@1+ (1,0) [0|1] "" Vector__XXX

BO_ 1 Right_Lane_A: 8 Vector__XXX
 SG_ lane_heading : 48|16@1- (0.0009765625,0) [-0.357|0.357] "radians" Vector__XXX
 SG_ lane_curvature_derivative : 32|16@1- (3.72529029846191E-009,0) [-0.00012|0.00012] "1/m^2" Vector__XXX
 SG_ lane_curvature : 16|16@1- (9.765625E-007,0) [-0.02|0.02] "1/m" Vector__XXX
 SG_ distance_to_lane : 0|16@1- (0.00390625,0) [-127|128] "meter" Vector__XXX

BO_ 0 Left_Lane_A: 8 CAVs_System
 SG_ lane_heading : 48|16@1- (0.0009765625,0) [-0.357|0.357] "radians" Vector__XXX
 SG_ lane_curvature_derivative : 32|16@1- (3.72529029846191E-009,0) [-0.00012|0.00012] "1/m^2" Vector__XXX
 SG_ lane_curvature : 16|16@1- (9.765625E-007,0) [-0.02|0.02] "1/m" Vector__XXX
 SG_ distance_to_lane : 0|16@1- (0.00390625,0) [-127|128] "meter" Vector__XXX



BA_DEF_  "BusType" STRING ;
BA_DEF_ BU_  "NodeLayerModules" STRING ;
BA_DEF_ BU_  "ECU" STRING ;
BA_DEF_ BU_  "CANoeJitterMax" INT 0 0;
BA_DEF_ BU_  "CANoeJitterMin" INT 0 0;
BA_DEF_ BU_  "CANoeDrift" INT 0 0;
BA_DEF_ BU_  "CANoeStartDelay" INT 0 0;
BA_DEF_DEF_  "BusType" "";
BA_DEF_DEF_  "NodeLayerModules" "";
BA_DEF_DEF_  "ECU" "";
BA_DEF_DEF_  "CANoeJitterMax" 0;
BA_DEF_DEF_  "CANoeJitterMin" 0;
BA_DEF_DEF_  "CANoeDrift" 0;
BA_DEF_DEF_  "CANoeStartDelay" 0;
BA_ "BusType" "CAN";

